A bunch of small, hacky improvements for the hackathon

Schwartz10 opened the pull request.

Schwartz10 committed:
A bunch of small, hacky improvements for the hackathon

Schwartz10 committed:
Fix event logs req getting cid

Schwartz10 committed:
2.1.6-0

Schwartz10 committed:
2.1.6-1

navFooh committed:
Update FEVM improvements

update filecoin-actor-utils

Update IconCloseStyled transition

Use useCallback in useLocalStorage

Add comments to useLocalStorage

Update useAbi.ts

Update useAbi.ts

Update useAbi.ts

Update AbiSelector

update packages

navFooh commented on src/components/Layout/LinesReturn.tsx:
```diff
@@ -103,6 +103,8 @@ export const LinesReturn = ({
     <DataTypeLines label='Return' dataType={describedReturnVal} />
   ) : isDelegated && !abi ? (
     <Line label='Return (upload abi to decode)'>{returnVal}</Line>
+  ) : isDelegated ? (
+    <NullishLine label='Return' />
```
@Schwartz10 I'm not sure why we should add the `NullishLine` here? If the actor is delegated and we do have the ABI, then it should either parse to a `describedReturnVal` or we should show that it failed to decode with the raw `returnVal`. Showing a `NullishLine` just makes it seem like there is not return value altogether.

navFooh commented on src/components/Layout/LinesParams.tsx:
```diff
@@ -99,14 +99,10 @@ export const LinesParams = ({ address, method, params }: LinesParamsProps) => {
-    >
-      {params}
-    </Line>
+  ) : params && decodeParamsError ? (
+    <Line label='Params (failed to decode)'>{params}</Line>
+  ) : params && !isDelegated ? (
+    <Line label='Params (decoding...)'>{params}</Line>
```
I think this seems to prevent a `(decoding...)` state for the params when the actor is delegated, which might make sense because parameter decoding is synchronous. But I think it also causes to now just show the `NullishLine` if the parameters for a smart contract tx fail to decode for some reason.

navFooh committed:
Update FEVM improvements 2

Delete TxID.graphql

Linter fix

Remove linter error

Update tx id logic

Update index.ts

Update index.tsx

navFooh committed:
update msg detail tx ID

navFooh committed:
Update useAbi.ts

navFooh committed:
Remove swr config for immutable

navFooh committed:
Update useIsFEVMActor.ts

navFooh committed:
useGetFEVMLogs

navFooh committed:
Update useIsFEVMActor.ts

navFooh committed:
Move useIsFEVMActor

navFooh committed:
Add util exports

navFooh committed:
Update LinesReturn.tsx

navFooh committed:
update snapshot

navFooh committed:
Update index.ts

navFooh committed:
Update LinesParams.tsx

navFooh committed:
Update LinesEventLogs.tsx

navFooh committed:
contractName

navFooh committed:
Update LinesParams.tsx

navFooh committed:
Update LinesReturn.tsx

navFooh committed:
prevent proptype error

navFooh committed:
LinesFEVMLogs

navFooh committed:
Update LinesFEVMLogs.tsx

navFooh committed:
Specify coinType for delegatedFromEthAddress

navFooh committed:
Add missing hook dependency

navFooh committed:
Don't show error when described FEVM params result to null

navFooh committed:
Update LinesFEVMLogs.tsx

navFooh committed:
Merge branch 'primary' into fevm-improvements
